site/pages/what-we-do/index.js

```diff
@@ -86,3 +76,22 @@ export default function whatWeDo() {
     </div>
   );
 }
+
+export default function whatWeDo() {
+  return (
+    <div>
+      <div className={styles.pageHeaderContainer}>
+        <h1 className={styles.pageHeader}>
```

Comment:
Can we combine these 2 h1:s into one, with spans? Otherwise screen reader splits them and it's become less readable.